portal-ui/src/screens/Console/Dashboard/Prometheus/PrDashboard.tsx

```diff
@@ -122,9 +132,9 @@ const PrDashboard = ({
                   key={`widget-${panelInline}-${indexPanel}`}
                   className={classes.widgetPanelDelimiter}
                   item
-                  xs={7}
-                  sm={8}
-                  md={6}
+                  xs={12}
+                  sm={perc as GridSize}
```

Comment:
There is an issue with sm size, can we make it 12 as well please?

<img width="337" alt="Screen Shot 2021-11-04 at 13 05 29" src="https://user-images.githubusercontent.com/33497058/140412107-13c6ca07-d66f-4b6b-b8f0-ecfc7db9e7a3.png">

```suggestion
                  xs={12}
                  sm={12}
```